dd-sdk-android-core/src/main/kotlin/com/datadog/android/v2/core/SdkInternalLogger.kt

```diff
@@ -30,38 +30,44 @@ internal class SdkInternalLogger(
 ) : InternalLogger {
 
     /**
-     * Global Dev Logger. This logger is meant for user's debugging purposes.
+     * This logger is meant for user's debugging purposes.
      * Logcat logs are conditioned by the [Datadog.libraryVerbosity].
      * No Datadog logs should be sent.
      */
-    internal val devLogger = devLogHandlerFactory.invoke()
+    internal val userLogger = userLogHandlerFactory.invoke()
 
     /**
-     * Global SDK Logger. This logger is meant for internal debugging purposes.
+     * This logger is meant for internal debugging purposes.
      * Logcat logs are conditioned by a BuildConfig flag (set to false for releases).
      */
-    internal val sdkLogger = sdkLogHandlerFactory.invoke()
+    internal val maintainerLogger = maintainerLogHandlerFactory.invoke()
+
+    private val onlyOnceUserMessages = mutableListOf<String>()
+    private val onlyOnceMaintainerMessages = mutableListOf<String>()
+    private val onlyOnceTelemetryMessages = mutableListOf<String>()
```

Comment:
should it be `mutableSetOf` instead? otherwise we are hitting O(N) for `contains` calls.

Reply:
Good call